boards/clue_nrf52840/README.md

```diff
@@ -0,0 +1,99 @@
+Adafruit CLUE - nRF52840 Express with Bluetooth LE
+==================================================
+
+<img src="https://cdn-learn.adafruit.com/assets/assets/000/087/843/medium640/adafruit_products_Clue_top_angle.jpg?1580406577" width="35%">
+
+The [Adafruit CLUE - nRF52840 Express with Bluetooth LE](https://www.adafruit.com/product/4500) is a
+board based on the Nordic nRF52840 SoC. It includes the
+following sensors:
+
+- 9 axis inertial LSM6DS33 + LIS3MDL sensor 
+- humidity and temperature sensor
+- barometric sensor
+- microphone
+- gesture, proximity, light color and light intensity sensor
+
+It has the same form factor with BBC:Microbit
+
+## Getting Started
+
+First, follow the [Tock Getting Started guide](../../../doc/Getting_Started.md)
+
+You will need the bossac bootloader tool:
+
+```shell
+$ pip3 install --user adafruit-nrfutil
+```
+
+## Programming the Kernel
+
+To program the kernel we use the Adafruit version of nRFUtil tool to communicate with the bootloader
+on the board which then flashes the kernel. This requires that the bootloader be
+active. To force the board into bootloader mode, press the button on the back of the board
+twice in rapid succession. You should see the red LED pulse on and off.
+
+At this point you should be able to simply run `make program` in this directory
+to install a fresh kernel.
+
+```
+$ make program
+```
+
+You may need to specify the port like so:
+
+```
+$ make program PORT=<serial port path>
+```
+
+## Programming Applications
+
+For now, flashing apps is only available together with the kernel
+
+```
+$ git clone https://github.com/tock/libtock-c
+$ cd libtock-c/examples/blink
+$ make
+```
+
+This previous step will create a TAB (`.tab` file) that normally tockloader
+would use to program on the board. However, tockloader is currently not
+supported.
+
+### Userspace Resource Mapping
+
+This table shows the mappings between resources available in userspace
+and the physical elements on the Nano 33 BLE board.
+
+| Software Resource | Physical Element    |
+|-------------------|---------------------|
+| GPIO[2]           | Pin D2              |
+| GPIO[3]           | Pin D3              |
+| GPIO[4]           | Pin D4              |
+| GPIO[6]           | Pin D6              |
+| GPIO[7]           | Pin D7              |
+| GPIO[8]           | Pin D8              |
+| GPIO[9]           | Pin D9              |
+| GPIO[10]          | Pin D10             |
+| GPIO[12]          | Pin D12             |
+| LED[0]            | Red LED Red         |
+| LED[1]            | White LEDs          |
```

Comment:
Is this correct?

Reply:
Yes, LED[1] is connected to two white LEDs on the front side